validation/validation.go

```diff
@@ -14,7 +14,7 @@ import (
 
 const defaultScheme = "http"
 
-var supportedProtocol = [2]string{"http", "https"}
+var supportedProtocol = [2]string{"http", "https", "rdp"}
```

Comment:
`[2]` should probably just be `[]`, and it would be nice if `supportedProtocol` was called `supportedProtocols` (plural) at this point

Reply:
Fixed both.